src/http/httpClient.ts

```diff
@@ -91,14 +91,14 @@ export class HttpClient {
       }
 
       newRequest.data = form;
-      mergeHeaders(newRequest.headers, form.getHeaders());
+      mergeHeaders(newRequest.headers || {} , form.getHeaders());
```

Comment:
Just realized we could have updated/declared `newRequest.headers` once on line 65 instead of three places!